src/careamics/models/lvae/noise_models.py

```diff
@@ -34,20 +34,76 @@ def noise_model_factory(
         If the chosen noise model `model_type` is not implemented.
         Currently only `GaussianMixtureNoiseModel` is implemented.
     """
-    if model_config:
+    if model_config.noise_models:
         noise_models = []
-        for nm_config in model_config.noise_models:
-            if nm_config.model_type == "GaussianMixtureNoiseModel":
-                noise_models.append(GaussianMixtureNoiseModel(nm_config))
-            else:
-                raise NotImplementedError(
-                    f"Model {nm_config.model_type} is not implemented"
-                )
+        for nm in model_config.noise_models:
+            if nm.path:
+                if nm.model_type == "GaussianMixtureNoiseModel":
+                    noise_models.append(GaussianMixtureNoiseModel(nm))
+                else:
+                    raise NotImplementedError(
+                        f"Model {nm.model_type} is not implemented"
+                    )
+
+            else:  # TODO this means signal/obs are provided. Controlled in pydantic model
+                # TODO train a new model. Config should always be provided?
+                if nm.model_type == "GaussianMixtureNoiseModel":
```

Comment:
Small detail: checking this conditional before checking `nm.path` will avoid code duplication :)